lib/clamav/connection.rb

```diff
@@ -32,16 +34,38 @@ def initialize(args)
 
     def establish_connection
       wrapped_request = @wrapper.wrap_request("IDSESSION")
+
+      puts "Writing request: #{wrapped_request}" if DEBUG
```

Comment:
I'd rather implement https://github.com/franckverrot/clamav-client/issues/4, I think.  Log levels ought to be changed dynamically and the use of a logger is preferable.